### PR TITLE
add airswap pairs and ticker

### PR DESCRIPTION
**Pull request by wenjieyek**
- What is the purpose of this Pull Request?
- What is the related issue for this Pull Request (if this PR fixes issue, prepend with "Fixes" or "Closes")?
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly

### CODE REVIEW

**Comment by tmlee** [comment]
https://github.com/coingecko/cryptoexchange/issues/926

File: lib/cryptoexchange/exchanges/airswap/services/market.rb [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+          adapt_all output
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::Airswap::Market::API_URL}/ticker"
+        end
+
+        def adapt_all(output)
+          output.map do |pair, output|
+            base, target = pair.split('_')
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/airswap/services/market.rb]: it looks like airswap returns pairs like ETH_DAI, ETH_FUN

ETH is the target, DAI is the base
need to reverse the assignment

File: README.md [hunk 2]
```diff
@@ -29,7 +29,8 @@ Or install it yourself as:
 | Abucoins          | Y       | Y          | Y       |         | Y           |          | abucoins          |
 | ACX               | Y       | Y          | Y       |         | Y           | Y        | acx               |
 | AEX               | Y       | Y          | Y       |         | Y           |          | aex               |
-| Allcoin           | Y       | Y          | Y       |         | Y           |          | allcoin           |
+| Airswap           | Y       |            |         |         | Y           |          | airswap           |
+| Allcoin           | Y       | Y          |         |         | User-Defined|          | allcoin           |
```
Comment (wongy91) [inline README.md]: please discard non-related change. Rebase your branch against master

File: README.md [hunk 3]
```diff
@@ -166,7 +167,7 @@ Or install it yourself as:
 | Gemini            | Y       | Y          | Y       |         | Y           |          | gemini            |
 | GetBTC            | Y       | Y          | Y       |         | User-Defined| Y        | getbtc            |
 | Gobaba            | Y       |            |         |         | Y           |          | gobaba            |
-| Gibraltar         | Y       | N          | N       |         | Y           | Y        | gbx               |     
+| Gibraltar         | Y       | N          | N       |         | Y           | Y        | gbx               |
```
Comment (wongy91) [inline README.md]: same here

File: README.md (hunk 2 above)
Comment (wenjieyek) [reply README.md]: erm I already rebase onto master, am I right? here I add the airswap to readme, so it considers related?

Comment (wongy91) [reply README.md]: Allcoin should have no changes.

Comment (wenjieyek) [reply README.md]: oh I see

**Comment by wenjieyek** [comment]
@wongy91 please see the file changes, am I right?

File: lib/cryptoexchange/exchanges/airswap/services/market.rb [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+          ticker           = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = Airswap::Market::NAME
+
+          ticker.volume    = NumericHelper.to_d(output['baseVolume'])
+          ticker.last      = NumericHelper.to_d(output['last'])
+          ticker.high      = NumericHelper.to_d(output['high24hr'])
+          ticker.low       = NumericHelper.to_d(output['low24hr'])
+          ticker.timestamp = Time.now.to_i
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/airswap/services/market.rb]: We no longer want to set Time.now.to_i for API that do not return time. Lets omit this

**Pull request merged**